lib/src/fade.dart

```diff
@@ -117,22 +124,19 @@ class _FadeTextState extends State<FadeAnimatedTextKit>
 
   int _currentRepeatCount;
 
-  Duration _duration;
-
   @override
   void initState() {
     super.initState();
 
-    _pause = widget.pause ?? const Duration(milliseconds: 500);
-
     _index = -1;
 
     _currentRepeatCount = 0;
 
-    _duration = widget.duration ?? const Duration(milliseconds: 2000);
-
     widget.text.forEach((text) {
-      _texts.add({'text': text, 'pause': _pause});
+      _texts.add({
+        'text': text,
+        'pause': widget.pause,
+      });
```

Comment:
Idk why do we need this anyways pause is constant..?

Reply:
YES!  You're right!  The refactoring helps make this crystal clear.

I'll go one step further...  Why should `_texts` exist?  It should simply reference `widget.text`.

I did not remove it because I wondered if there was a grand agenda to its existence. (🤔 I often wonder, _"What was the original author thinking?"_)

I'm happy to reconsider removing it, but I would prefer doing it as a follow-up PR because it will impact other classes and the recent emoji-related changes may need to be tweaked.  (I like to avoid too many ideas in a PR because it makes it more challenging to understand what was changed and why.)

Reply:
Yaa I also felt that but I haven't included it WavyAnimatedTexKit, maybe it was a thing that all the authors that it might have some use can but never had one 🤔 @aagarwal1012 do you have any idea?